live/global.ini

```diff
@@ -46094,7 +46094,7 @@ hurston_hackprevent_desc_001=HURSTON DYNAMICS\nOUTSOURCING DEPARTMENT\n---------
nicht vertraulich ist und die darin enthaltenen Informationen nach Bedarf offengelegt werden können, um die Erfüllung der vertraglich vereinbarten Aufgaben zu gewährleisten.\n\n15) Geltendes Recht: Dieser Vertrag unterliegt den Gesetzen des Handelsterritoriums des Planeten Hurston, Stanton System, United Empire of Earth, und wird in Übereinstimmung mit diesen Gesetzen ausgelegt, unter voller Berücksichtigung der Rechtswahlgesetze, und alle Streitigkeiten, die im Zusammenhang mit diesem Vertrag entstehen, werden nach diesen Gesetzen behandelt, einschließlich der geltenden Verjährungsgesetze.\n\nDu erklärst dich mit der ausschließlichen Zuständigkeit der Gerichte von Hurston und der ausschließlichen Zuständigkeit und dem Gerichtsstand der dortigen Gerichte einverstanden. Der/die unabhängige Auftragnehmer/in ist nicht verpflichtet, persönlich zu erscheinen, um an einem Verfahren teilzunehmen, das im Rahmen dieses Vertrages angestrengt wird, und kann per Fernkonferenz und/oder Korrespondenz teilnehmen, wie es unter den Umständen angemessen ist. \n\nFür den Fall, dass das anwendbare Recht ausdrücklich vorschreibt, dass eine Angelegenheit an die Gilde des unabhängigen Schiedsrichters zu richten ist, vereinbaren die Parteien, dass die Durchführung eines solchen Konfliktlösungsverfahrens per Fernzugriff erfolgen kann, ohne dass es zu einer Verlegung oder einem Wechsel des Gerichtsstands aus dem Handelsgebiet des Planeten Hurston kommt.\n\n16) Allgemeine Bedingungen: Alle Materialien, die das Unternehmen im Rahmen des Vertrags von oder im Namen des Unternehmens erhalten hat, müssen in demselben Zustand, in dem sie erhalten wurden, an eine Betriebszentrale des Unternehmens zurückgegeben werden.\n\nAndernfalls kann es als Vertragsbruch und als gestohlen oder anderweitig veruntreut angesehen werden; das Unternehmen behält sich alle Rechte nach Recht und Billigkeit vor, Maßnahmen und Aktionen zu ergreifen, die erforderlich sind, um jegliches Eigentum - physisch, materiell, geistig oder anderweitig - wiederzuerlangen. Die Parteien sind sich einig, dass der Vertrag und alle damit verbundenen Bestimmungen keinen Knebelvertrag darstellen. Die Parteien sind sich einig, dass der Vertrag und alle Änderungen dazu die vollständige Vereinbarung zwischen dem Unternehmen und dem/der unabhängigen Auftragnehmer/in darstellen, ohne dass auf außervertragliches Beweismaterial zurückgegriffen werden muss.\n\nDieser Vertrag ist im größtmöglichen gesetzlich zulässigen Umfang durchsetzbar; sollte ein Teil des Vertrags als nicht durchsetzbar angesehen werden, wird dieser Teil gestrichen und der Rest des Vertrags bleibt in vollem Umfang in Kraft, wobei alle verbleibenden Unklarheiten zugunsten der Absichten des Unternehmens ausgelegt werden, die am besten durch den Rückgriff auf externe Beweise, falls erforderlich, ermittelt werden.Nach Abschluss oder Beendigung des Vertrags bleiben die Abschnitte 5, 8, 9, 11, 14 und 15 unbefristet in Kraft. Trennbarkeit und Gesamtvereinbarung.\n\n
```

Comment:
```suggestion
hurston_intro_desc=HURSTON DYNAMICS\nACHTUNG ZUKUENFTIGE HURSTON-AUFTRAGNEHMER:\n----------------------------\n\nBevor eine Beschäftigung durch die Hurston Dynamics Outsourcing-Abteilung möglich ist, müssen alle selbstständigen Auftragnehmer zunächst die in der folgenden Vereinbarung für selbstständige Auftragnehmer aufgeführten Bedingungen akzeptieren.\n\n\n--- UNABHÄNGIGER AUFTRAGNEHMERVERTRAG ---\n\nDieser Vertrag für unabhängige Auftragnehmer (der "Vertrag") wird zwischen Hurston Dynamics ("Unternehmen") und dir ("unabhängiger Auftragnehmer") in Bezug auf die vom Unternehmen angebotene und von dir über dein mobiGlas angenommene Arbeit ("vertraglich vereinbarte Aufgabe")* geschlossen und tritt zu dem Zeitpunkt in Kraft, an dem deine Annahme per Handshake-Empfangsprotokoll vom Logistikdatenzentrum des Unternehmens empfangen und bestätigt wird.\n\n1) Arbeitsbeschreibung: Der unabhängige Auftragnehmer verpflichtet sich, alle Anforderungen, die in der Ausschreibung der vertraglichen Aufgabe angegeben sind, vollständig zu erfüllen. Der/die unabhängige Auftragnehmer/in erklärt sich damit einverstanden und erkennt an, dass ihm/ihr keine Zahlung, Rückerstattung oder sonstige Entschädigung vom Unternehmen zusteht, wenn nicht alle beschriebenen Anforderungen zur Zufriedenheit des Unternehmens erfüllt werden, was im alleinigen Ermessen des Unternehmens liegt.\n\n		a. Der unabhängige Auftragnehmer erklärt sich damit einverstanden und erkennt an, dass alle Bedingungen, einschließlich des Zeitplans, des Lieferdatums, der Identifizierung des Ziels, des Zustands der Waren oder anderer Umstände, die das Unternehmen für relevant hält, für diesen Vertrag wesentlich sind und dass dem unabhängigen Auftragnehmer keine anteiligen Zahlungen, Rückerstattungen, Ausgaben, Teilvergütungen oder andere Zahlungen für eine teilweise Erfüllung der vertraglich vereinbarten Aufgabe zustehen.\n\n		b. Der/die unabhängige Auftragnehmer/in erklärt sich damit einverstanden und erkennt an, dass dieser Vertrag ein eigenständiger Vertrag ist, der dem Unternehmen keine impliziten Verpflichtungen oder Haftungen aufgrund anderer Verträge auferlegt, die zwischen dem Unternehmen, dem/der unabhängigen Auftragnehmer/in oder einem Dritten bestehen.\n\n		c. Der/die unabhängige Auftragnehmer/in verpflichtet sich, keine anderen Arbeiten oder Aufgaben zu übernehmen, die die rechtzeitige Erledigung aller in der Leistungsbeschreibung und/oder in den Änderungen gemäß Abschnitt 3) des Vertrags genannten Aufgaben gefährden würden.\n\n		d. Das Unternehmen ist nicht für Fehler in der Genauigkeit verantwortlich, unabhängig davon, ob diese durch Mängel in der Datenübertragung, durch den Verlust der Signaltreue, durch Abfangen oder die Veränderung durch Dritte, einschließlich mobiGlas und anderer microTech-Produkte, verursacht wurden. Jegliche Zweifel an der Übereinstimmung zwischen dem Angebot des Unternehmens und seiner Lieferung sind im Hinblick auf die tatsächlichen Absichten des Unternehmens zu klären. Du erklärst dich damit einverstanden, alle Risiken zu übernehmen, die mit dem Empfang von Übertragungen oder Informationsübertragungen verbunden sind, die nicht direkt vom Unternehmen stammen.\n\n		e. Die Parteien sind sich darüber einig, dass die Erlöse und Entschädigungen aus dem Vertrag zwar einem Dritten zugutekommen können, dass jedoch kein Dritter berechtigt ist, rechtliche Schritte in Bezug auf den Vertrag einzuleiten, und dass jede falsche Darstellung seitens des unabhängigen Auftragnehmers in Abweichung von dieser Anforderung als Verstoß gegen seine Verpflichtungen und Verantwortlichkeiten im Rahmen dieses Vertrags angesehen werden kann.\n\n		f. Im Falle eines Widerspruchs zwischen den in der vertraglichen Aufgabe angegebenen Bedingungen und dieser Vereinbarung gelten die Bedingungen dieser Vereinbarung, es sei denn, die Bedingungen der vertraglichen Aufgaben berufen sich ausdrücklich auf Abschnitt 3 Punkt c. dieser Vereinbarung.\n\n2) Laufzeit: Die Laufzeit des Vertrags ist in der Leistungsbeschreibung angegeben und endet in jedem Fall mit der Zahlung gemäß Abschnitt 13) des Vertrags oder mit dem endgültigen Abschluss aller Aufgaben im Rahmen der Leistungsbeschreibung, je nachdem, was später eintritt.\n\n3) Änderungen: Die Parteien vereinbaren, dass das Unternehmen die Leistungsbeschreibung während der Ausführung der Arbeiten ändern kann, um auf betriebliche Unwägbarkeiten oder dringende Anforderungen zu reagieren. Der unabhängige Auftragnehmer kann die Änderungen akzeptieren, indem er dies in einer elektronisch signierten Handshake-Freigabeübermittlung angibt.\n\nWenn der unabhängige Auftragnehmer das Unternehmen nicht innerhalb von 24 Stunden benachrichtigt, betrachten die Parteien die Änderung als angenommen und die Bedingungen der Leistungsbeschreibung werden entsprechend geändert.\n\n		a. Wenn der/die unabhängige Auftragnehmer/in die Änderungen nicht akzeptieren möchte, besteht sein/ihr einziger Ausweg darin, den Vertrag zu kündigen und alle Materialien des Unternehmens (einschließlich Informationen oder Daten) und/oder alle anderen Materialien in die Obhut des Unternehmens zurückzugeben.\n\n		b. Vertragliche Aufhebung durch Änderung; wie in Abschnitt 1) 3. erwähnt, können die Bestimmungen dieser Vereinbarung durch die Bestimmungen der vertraglich vereinbarten Aufgabe oder einer Änderung ersetzt werden, wenn dies ausdrücklich unter Bezugnahme auf den entsprechenden Abschnitt geltend gemacht wird.\n\n4) Ausgaben und Rückerstattungen\n\n		a. Die Nichteinhaltung des erforderlichen Versicherungsschutzes gilt als wesentliche Nichterfüllung der Verantwortlichkeiten und Verpflichtungen des unabhängigen Auftragnehmers im Rahmen des Vertrags. Der/die unabhängige Auftragnehmer/in ist verpflichtet, die folgenden Versicherungen als Voraussetzung für die Ausführung von Arbeiten für das Unternehmen abzuschließen:\n\n		a 1. Kaskoversicherung für alle Fahrzeuge, die bei der Erfüllung der Aufgabe eingesetzt werden.\n\n		a 2. Frachtversicherung für jegliches Material, das von, für oder im Namen des Unternehmens gemäß dem Vertrag erhalten wird.\n\n		a 3. Versicherung von Ausrüstung und Geräten für alle Ausrüstungsgegenstände, die das Unternehmen im Rahmen des Vertrags erhalten hat oder die für das Unternehmen bestimmt sind.\n\n		b. Der/die unabhängige Auftragnehmer/in erkennt an, dass die Kosten für alle Ausgaben oder betrieblichen Aufwendungen, die für die Durchführung der vertraglich vereinbarten Aufgaben erforderlich sind, durch die in der vertraglich vereinbarten Aufgabe beschriebene Vergütung implizit abgedeckt sind. Das Unternehmen ist nicht verantwortlich für Kostenüberschreitungen, Schäden, Reparaturen oder andere Ausgaben, sei es in Form von Zeit oder Geld.\n\n		c. Die Kosten für die medizinische Versorgung sind ausdrücklich von den Verpflichtungen des Unternehmens aus diesem Vertrag ausgeschlossen. Alle Ausgaben im Zusammenhang mit der persönlichen Gesundheit des/der unabhängigen Auftragnehmers/in gelten als persönliche Ausgaben.\n\n5) Nichtverunglimpfung: Der/die unabhängige Auftragnehmer/in sichert dem Unternehmen zu, dass alle geschäftlichen oder persönlichen Aktivitäten, die er/sie gleichzeitig mit der beauftragten Aufgabe ausübt, nach den örtlichen Verordnungen und/oder den geltenden Gesetzen der UEE legal sind und sich nicht negativ auf den Ruf des Unternehmens auswirken oder diesen schädigen werden.\n\n		a. Der/die unabhängige Auftragnehmer/in erklärt sich ausdrücklich damit einverstanden, das Unternehmen von jeglicher Schuld oder Verantwortung im Zusammenhang mit behördlichen oder gerichtlichen Maßnahmen, Untersuchungen oder Ermittlungen freizustellen, die die Tätigkeit des/der unabhängigen Auftragnehmers/in betreffen.\n\n		b. Die Vertragsparteien vereinbaren, dass der/die Selbstständige das Unternehmen nicht verunglimpfen, herabsetzen, kritisieren oder auf andere Weise in Wort, Schrift, Spektrumübertragung oder Rezensionen negativ darstellen wird. Ein Verstoß gegen diese Klausel kann zum Entzug oder zur anteiligen Rückerstattung von Gebühren oder Gegenleistungen aus dem Vertrag führen.\n\n6) Keine Vertretung - Der unabhängige Auftragnehmer darf sich unter keinen Umständen als Vertreter, leitender Angestellter, Direktor, Mitarbeiter, Repräsentant oder sonstiges Mitglied des Unternehmens ausgeben. Die Beziehung des/der unabhängigen Auftragnehmers/in zum Unternehmen ist ausschließlich die eines/einer unabhängigen Auftragnehmers/in.\n\nDer/die unabhängige Auftragnehmer/in darf sich als unabhängiger Auftragnehmer ausgeben, der/die Arbeiten für das Unternehmen ausführt, indem er/sie die mit der vertraglich vereinbarten Aufgabe verbundene Logistics Registry Identification an eine Zivil- oder Strafverfolgungsbehörde weitergibt, und zwar ausschließlich zu dem Zweck, eine vorrangige Genehmigung für Reisen oder Sicherheitsmaßnahmen zu erhalten.\n\n7) Vergabe von Unteraufträgen - Du unterschreibst und stimmst diesem Vertrag im Namen aller Personen zu, die an der Ausführung der in der Arbeitsbeschreibung beschriebenen Aufgaben beteiligt sind. Die Auswahl der Subunternehmer unterliegt der Zustimmung des Unternehmens; du übernimmst die Verantwortung für alle Verbindlichkeiten und Risiken, die sich aus dem Einsatz von nicht genehmigten Subunternehmern ergeben, und zwar über die in diesem Vertrag festgelegten Strafen hinaus.\n\nWenn ein nicht zugelassener Subunternehmer eine Vertragsverletzung begeht, ist das Unternehmen nicht verpflichtet, daraus resultierende Schäden oder Folgeschäden zu mindern.\n\n8) Schadloshaltung und Haftung: Der unabhängige Auftragnehmer erklärt sich damit einverstanden, das Unternehmen und alle mit ihm verbundenen Unternehmen von jeglichem Verlust, jeglicher Haftung, jeglichem Schaden und jeglichem Folgeschaden freizustellen, zu verteidigen und schadlos zu halten, einschließlich und ohne Einschränkung jeglicher Anwaltskosten, die dem unabhängigen Auftragnehmer infolge der Erfüllung der vertraglich vereinbarten Aufgabe entstehen.\n\nDer/die unabhängige Auftragnehmer/in erklärt sich außerdem bereit, das Unternehmen und alle mit ihm verbundenen Unternehmen von jeglichen Ansprüchen auf Schadenersatz, Tod oder Verstümmelung, Rufschädigung, Verletzung von materiellem, geistigem, physischem oder anderem Eigentum, strafrechtlichen Anklagen oder sonstigen Ansprüchen nach Recht und Billigkeit freizustellen, die von Dritten im Zusammenhang mit der Ausführung einer vertraglich vereinbarten Aufgabe durch den/die unabhängige/n Auftragnehmer/in erhoben werden.\n\nDie Parteien vereinbaren, dass etwaige Schäden, die das Unternehmen dem unabhängigen Auftragnehmer schuldet, ohne besondere, zufällige oder Folgeschäden sowie ohne Strafschadensersatz zu leisten sind und vollständig durch Gutschriften ausgeglichen werden können (einschließlich der Bereitstellung von Gutschriften des Unternehmens, die bei verbundenen Unternehmen des Unternehmens eingelöst werden können), und dass bestimmte Leistungen, Unterlassungsklagen und billigkeitsrechtliche Rechtsbehelfe in vollem Umfang ausgeschlossen sind.\n\n9) Besondere Leistung und gerechte Entschädigung: Die Parteien erkennen an, dass die Zeit ein wesentlicher Bestandteil des Vertrags und eine wesentliche Bestimmung der vertraglich vereinbarten Aufgabe ist und dass Schadenersatz in Geld allein das Unternehmen nicht für die Nichterfüllung des Vertrags entschädigen kann.\n\n10) Garantien und Gewährleistungen: Das Unternehmen übernimmt keine Garantien oder Gewährleistungen für Ausrüstungen, Leihgaben, Ausrüstungen oder andere logistische oder betriebliche Materialien, die vom unabhängigen Auftragnehmer verwendet oder eingesetzt werden, mit Ausnahme derjenigen, die vom Unternehmen oder einem seiner verbundenen Unternehmen hergestellt oder produziert wurden; in diesem Fall gelten die entsprechenden Herstellergarantien und/oder Haftungsausschlüsse nur für diese Artikel.\n\n11) Geistiges Eigentum: Ungeachtet des Abschnitts 6) erklärst du dich damit einverstanden, dass alle Erfindungen oder Innovationen, die du während der Ausführung der vertraglich vereinbarten Aufgabe unter Verwendung von durch das Unternehmen zur Verfügung gestellten Materialien erdacht, ermöglicht oder in die Praxis umgesetzt hast, als "Auftragsarbeiten" gelten, die vom Unternehmen verfasst wurden und dessen Eigentum sind, soweit dies gesetzlich zulässig ist.\n\nIn Gerichtsbarkeiten, die solche Vereinbarungen nicht zulassen, erklärst du dich bereit, die Urheberschaft und das Eigentum an solchen Erfindungen oder Geschäftsgeheimnissen an das Unternehmen abzutreten und mit dem Unternehmen bei der Erteilung von Genehmigungen, Erlaubnissen oder Bestätigungen zusammenzuarbeiten, die für die rechtliche Abtretung solcher Eigenschaften erforderlich sind.\n\n		a. Im Falle eines inhärenten Miteigentums an solchen Erfindungen oder Innovationen hat das Unternehmen im gleichen Maße wie jeder andere Teilnehmer oder Mitwirkende an der Innovation ein gemeinsames Eigentum und Interesse. Das Unternehmen ist nicht verpflichtet, Informationen oder sonstiges geistiges Eigentum mit solchen Mitpächtern zu teilen.\n\n		b. Der unabhängige Auftragnehmer verpflichtet sich, alle erforderlichen Genehmigungen oder Technologielizenzen einzuholen, die für die Durchführung der vertraglich vereinbarten Aufgabe erforderlich sind.\n\n		c. Der/die unabhängige Auftragnehmer/in erklärt sich damit einverstanden, dass das Unternehmen im Falle eines Vertragsbruchs aufgrund der Nichterfüllung der kontaktierten Aufgabe rechtlich befugt ist, Zugang zu den Schiffs- und mobiGlas-Logbüchern des/der unabhängigen Auftragnehmers/in, zu seinen/ihren persönlichen Gegenständen, zu den Blackbox-Recordern und zu den an Bord befindlichen oder persönlichen Datenspeichern zu erhalten, um das Ausmaß der Nichterfüllung und/oder alle Umstände festzustellen, die zu einer Milderung oder Verschlimmerung des daraus resultierenden Schadens für das Unternehmen führen könnten.\n\n12) Höhere Gewalt: Der/die unabhängige Auftragnehmer/in erklärt sich damit einverstanden und erkennt an, dass einige der Verpflichtungen oder Haftungen des Unternehmens im Rahmen dieses Vertrags von betrieblichen Erfordernissen oder anderweitig von Umständen abhängig sein können, die sich der Kontrolle des Unternehmens entziehen, und dass solche Verpflichtungen oder Haftungen des Unternehmens aufgrund der Unmöglichkeit oder der Verneinung oder des Wegfalls ihrer Geschäftszwecke nach alleinigem Ermessen des Unternehmens entfallen können.\n\n13) Rechnungsstellung: Nach Erfüllung aller in der Leistungsbeschreibung und den entsprechenden Änderungen genannten Anforderungen wird das Unternehmen die in der Leistungsbeschreibung angegebene Vergütung innerhalb von 30 Standardarbeitstagen per Post, Überweisung oder auf andere Weise an den unabhängigen Auftragnehmer übermitteln.\n\nWenn der/die unabhängige Auftragnehmer/in oder seine/ihre Konten während dieser Zeit nicht erreichbar sind, wird die Vergütung vom Unternehmen auf einem Treuhandkonto verwahrt, bis der/die unabhängige Auftragnehmer/in das Unternehmen benachrichtigt, dass er/sie bereit ist, die Vergütung zu erhalten, vorbehaltlich geltender Verjährungs- oder Abtretungsbestimmungen.\n\n14) Vertraulichkeit - Der/die unabhängige Auftragnehmer/in erklärt sich damit einverstanden, dass die gesamte Kommunikation mit dem Unternehmen und/oder seinen verbundenen Unternehmen, einschließlich seiner/ihrer Angestellten, leitenden Angestellten, Direktoren, Agenten oder Vertreter, streng vertraulich behandelt wird.\n\nDer/die unabhängige Auftragnehmer/in erklärt sich damit einverstanden, alle Protokolle der Kommunikation mit den Vorgenannten auf Verlangen des Unternehmens zu löschen oder zu bereinigen und diese Löschung auf Anfrage zu bestätigen. Die Parteien sind sich einig, dass die eingestellte Aufgabe nicht vertraulich ist und die darin enthaltenen Informationen nach Bedarf offengelegt werden können, um die Erfüllung der vertraglich vereinbarten Aufgaben zu gewährleisten.\n\n15) Geltendes Recht: Dieser Vertrag unterliegt den Gesetzen des Handelsterritoriums des Planeten Hurston, Stanton System, United Empire of Earth, und wird in Übereinstimmung mit diesen Gesetzen ausgelegt, unter voller Berücksichtigung der Rechtswahlgesetze, und alle Streitigkeiten, die im Zusammenhang mit diesem Vertrag entstehen, werden nach diesen Gesetzen behandelt, einschließlich der geltenden Verjährungsgesetze.\n\nDu erklärst dich mit der ausschließlichen Zuständigkeit der Gerichte von Hurston und der ausschließlichen Zuständigkeit und dem Gerichtsstand der dortigen Gerichte einverstanden. Der/die unabhängige Auftragnehmer/in ist nicht verpflichtet, persönlich zu erscheinen, um an einem Verfahren teilzunehmen, das im Rahmen dieses Vertrags angestrengt wird, und kann per Fernkonferenz und/oder Korrespondenz teilnehmen, wie es unter den Umständen angemessen ist. \n\nFür den Fall, dass das anwendbare Recht ausdrücklich vorschreibt, dass eine Angelegenheit an die Gilde des unabhängigen Schiedsrichters zu richten ist, vereinbaren die Parteien, dass die Durchführung eines solchen Konfliktlösungsverfahrens per Fernzugriff erfolgen kann, ohne dass es zu einer Verlegung oder einem Wechsel des Gerichtsstands aus dem Handelsgebiet des Planeten Hurston kommt.\n\n16) Allgemeine Bedingungen: Alle Materialien, die das Unternehmen im Rahmen des Vertrags von oder im Namen des Unternehmens erhalten hat, müssen in demselben Zustand, in dem sie erhalten wurden, an eine Betriebszentrale des Unternehmens zurückgegeben werden.\n\nAndernfalls kann es als Vertragsbruch und als gestohlen oder anderweitig veruntreut angesehen werden; das Unternehmen behält sich alle Rechte nach Recht und Billigkeit vor, Maßnahmen und Aktionen zu ergreifen, die erforderlich sind, um jegliches Eigentum - physisch, materiell, geistig oder anderweitig - wiederzuerlangen. Die Parteien sind sich einig, dass der Vertrag und alle damit verbundenen Bestimmungen keinen Knebelvertrag darstellen. Die Parteien sind sich einig, dass der Vertrag und alle Änderungen dazu die vollständige Vereinbarung zwischen dem Unternehmen und dem/der unabhängigen Auftragnehmer/in darstellen, ohne dass auf außervertragliches Beweismaterial zurückgegriffen werden muss.\n\nDieser Vertrag ist im größtmöglichen gesetzlich zulässigen Umfang durchsetzbar; sollte ein Teil des Vertrags als nicht durchsetzbar angesehen werden, wird dieser Teil gestrichen und der Rest des Vertrags bleibt in vollem Umfang in Kraft, wobei alle verbleibenden Unklarheiten zugunsten der Absichten des Unternehmens ausgelegt werden, die am besten durch den Rückgriff auf externe Beweise, falls erforderlich, ermittelt werden. Nach Abschluss oder Beendigung des Vertrags bleiben die Abschnitte 5, 8, 9, 11, 14 und 15 unbefristet in Kraft. Trennbarkeit und Gesamtvereinbarung.\n\n
```

Und bist du dir sicher, dass du die "unabhängigen Arbeitnehmer/innen" geschlechtsneutral machen willst?

Reply:
Gute Frage. Vielleicht nur in diesem einen Text?

Reply:
Ich finde, durch die oben genannte ```und dir ("unabhängiger Auftragnehmer")``` haben wir bereits gesetzt, dass wir den Spieler meinen. Was mich auch zu meiner 2. Frage dazu bringt. Sollen wir hier wirklich duzen? Das ist oben ja auch schon ein Vertrag, nicht?

Reply:
Ok ich ändere das fix mal um.